Streetcode/Streetcode.BLL/MediatR/Streetcode/Streetcode/GetAll/GetAllStreetcodesHandler.cs

```diff
@@ -20,13 +22,67 @@ public GetAllStreetcodesHandler(IRepositoryWrapper repositoryWrapper, IMapper ma
 
     public async Task<Result<IEnumerable<StreetcodeDTO>>> Handle(GetAllStreetcodesQuery request, CancellationToken cancellationToken)
     {
-        var streetcodes = await _repositoryWrapper.StreetcodeRepository.GetAllAsync();
+        var streetcodes = await _repositoryWrapper.StreetcodeRepository
+            .GetAllAsync();
 
         if (streetcodes is null)
         {
-            return Result.Fail(new Error($"Cannot find any streetcode"));
+            return Result.Fail(new Error($"Cannot find any streetcodes"));
         }
 
+        if (request.Title is not null)
```

Comment:
I'm little bit new to this repo and it's code, but don't you think that this method are too big and it can be optimized by splitting it to private methods?

Reply:
If you are talking about whole GetAllStreetcodes endpoint, It can be done this way, but this logic, I suppose, has to be used just in current endpoint, in oposite way, I prefer to create service and inject it, maybe it will be useful in other parts of project.

Reply:
splitting one big method in few small it's not only about reusability, but about readability and it makes it easy to understand

Reply:
Agree with you. And I suposse it is good idea. If logic of endpoint will be approved by Teachlead, I could change it that way (sepparate logic for different private methods).

Reply:
It was just suggesting, so final decision is up to you)